es-bs-base/scss/_buttons.scss

```diff
@@ -62,12 +64,14 @@ fieldset:disabled a.btn {
 // Alternate buttons
 //
 
+// DEPRECATED BUTTON STYLES
 @each $color, $value in $theme-colors {
```

Comment:
Wouldn't this result in these definitions still being part of the css ouput even if they are over written below? I think removing them is probably better.

Reply:
Yes, the definitions are still part of the CSS output. Whether to remove them or to keep them and mark them as deprecated was discussed during the last guild meeting, and to minimize potential disruption for people who might be using these styles, we decided to deprecate for now and remove them at a future time.